features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResourcesSerializer.kt

```diff
@@ -90,14 +99,15 @@ internal class Base64Serializer private constructor(
 
     // region private
 
+    @Suppress("ReturnCount")
     @WorkerThread
     private fun serializeBitmap(
```

Comment:
why is this called serializeBitmap ?

Reply:
This is renamed in the next PR to resolveResourceHash